### PR TITLE
[stable17] Revive the "send email to new users" toggle for the user form

### PR DESCRIPTION
**Pull request by ChristophWurst**
Backport of https://github.com/nextcloud/server/pull/19664

`@nextcloud/router` was not a thing at that time, hence using `OC.generateUrl`

Todo
- [x] Test

### CODE REVIEW

File: core/shipped.json [hunk 1]
```diff
@@ -49,6 +49,7 @@
     "lookup_server_connector",
     "provisioning_api",
     "oauth2",
+    "settings",
```
Comment (ChristophWurst) [inline core/shipped.json]: Without this change an `\OC\AppFramework\Middleware\Security\Exceptions\AppNotEnabledException` was thrown and I couldn't access the user settings.

Comment (rullzer) [reply core/shipped.json]: mmm so. you add stuff to apps/settings. But apps/settings didn't excist yet for 17 right?

Comment (ChristophWurst) [reply core/shipped.json]: Ugh. Yeah. It existed locally, but guess that was due to the build artifacts. Let me try to remove them.

Comment (ChristophWurst) [reply core/shipped.json]: Seems to work without this change now. No idea why, though.

**Review by rullzer — APPROVED** [review]

**Comment by rullzer** [comment]
CI is not happy... https://drone.nextcloud.com/nextcloud/server/26530/2/3

**Comment by kesselb** [comment]
@gary-kim are you the new rebase bot? ;)

**Pull request merged**